core/src/main/java/org/bitcoin/crawler/PeerSeedProtos.java

```diff
@@ -3,6 +3,7 @@
 
 package org.bitcoin.crawler;
 
+@SuppressWarnings("ALL")
```

Comment:
Yes, I'm afraid. But if the changes are committed manually, one might just leave the annotation in the file? I'm with if this is just not merged at all, though.